### DISCUSSION TITLE
Re-organize based on submodules?

### DISCUSSION THREAD

**Discussion by djsime1** [djsime1/awesome-flipperzero#41] [Polls]
Hey all, when I created this repository I never expected it to grow so large and even become the most stared Flipper Zero related repository. In that time, I've become aware that it can be quite a chore to read through everything, if you're looking for a resource pertaining to a particular core application.

So now I'm coming around to ask: Should I recategorize the list? I'm thinking that entries related to NFC, Sub-GHz, BadUSB, etc, would get put under their own categories, and everything else would remain where it is now. Give your answer on the poll below! Items are placed in order from least effort to most on my part.

Furthermore, I'm all for preservation, but more and more links are growing outdated, such as flooper-blooper. Should links like those be removed? If so, what would be the criteria for removal?

**Comment by Awethentique** [comment]
First and foremost. Thank you for starting something awesome here.

I would suggest if possible making all the linked resources as submodules to this repo. That way you could clone this repo and its submodules of the repos that you want to use and keep track of them as well when they update.

Perhaps there may be a better way, but this would be helpful
